### PR TITLE
test: update 0071-STAK ACs

### PR DESCRIPTION
**Pull request by C0deMunk33**

### CODE REVIEW

**Review by gordsport — DISMISSED** [review]

**Comment by gordsport** [comment]
@C0deMunk33 

During the protocol-design meeting we reviewed this and approved the removal. Before merging can you please confirm that the other ACs sufficiently covers the functionality of the  governance token staking

**Comment by gordsport** [comment]
@vega-paul @davidsiska-vega 
This has been updated to remove the invalid AC and adding further negative tests.

Work done on the smart contracts side is:
- https://github.com/vegaprotocol/system-tests/issues/1519

(kind of blocked until this is in and the CI is fixed)

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**
